docs/challenge-000.md

```diff
@@ -41,11 +39,19 @@ Next clone the repo locally and set up a dev container for your exi
 4. The VS Code window will reload and start building the dev container. A progress notification provides status updates. You only have to build a dev container the first time you open it; opening the folder after the first successful build will be much quicker.
 5. After the build completes, VS Code will automatically connect to the container.
 You can now interact with your project in VS Code just as you could when opening the project locally. From now on, when you open the project folder, VS Code will automatically pick up and reuse your dev container configuration.
+6. Build and run the starter code, setting a breakpoint to see how the 
```

Comment:
Seems like your sentence got cut off??

Reply:
yes, sorry wasn't finished with updates yet. should be fixed now.